build.xml

```diff
@@ -242,7 +242,8 @@
     <javadoc access="protected" author="false" classpathref="compile.classpath"
       destdir="${doc}" nodeprecated="false" nodeprecatedlist="false"
       noindex="false" nonavbar="false" notree="false"
-      source="11" splitindex="true" use="true" version="true">
+      source="11" splitindex="true" use="true" Windowtitle="Workspace 0.14.1 API"
```

Comment:
add in the window title to make the output from this command more similar to the settings that that the gradle javadoc task  uses.